publish to Hex.pm failed on v0.6.0

https://github.com/rclex/rclex/runs/5584771817?check_suite_focus=true

** (Mix) Stopping package build due to errors.
Dependencies excluded from the package (only Hex packages can be dependencies): rclex_gen_msgs
Error: Process completed with exit code 1.

--- 
we also need to publish https://github.com/rclex/rclex_gen_msgs to Hex.pm :D

--- a/mix.exs
+++ b/mix.exs
@@ -66,7 +66,7 @@ defmodule Rclex.MixProject do
       {:credo, "~> 1.5", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.0", only: [:dev], runtime: false},
       {:ex_doc, "~> 0.22", only: :dev, runtime: false},
-      {:rclex_gen_msgs, git: "https://github.com/rclex/rclex_gen_msgs.git"}
+      {:rclex_gen_msgs, "~> 0.1.0", runtime: false}
     ]
   end
 

